workers/unity/Packages/com.improbable.gdk.transformsynchronization/Systems/SetNewlyCreatedGameObjectTransformSystem.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Generated.Improbable;
+using Generated.Improbable.Transform;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Core.GameObjectRepresentation;
+using Unity.Collections;
+using Unity.Entities;
+using UnityEngine;
+using Quaternion = UnityEngine.Quaternion;
+
+namespace Improbable.Gdk.TransformSynchronization
```

Comment:
Feature modules shouldn't depend on each other like this if they can help it. As this only depends on core components there doesn't seem to be a reason this can't be with the other monobehaviour systems

Reply:
Correct, see above, will remove